### ISSUE TITLE
Crash on startup due to mismatched DPI

### ISSUE THREAD

**Issue by BreeceW** [DevToys-app/DevToys#1200]
### Current behavior

The app will open with a blank window and then crash before loading content.

### How to reproduce it (as minimally and precisely as possible)

1. Open DevToys-Preview from the Start menu
2. It crashes
3. After changing my system scaling (which is normally 125%) to 100%, it started working. On changing back to 125%, it now launches correctly.

### Expected behavior

The app should launch.

### Workaround

Change DPI in system settings and back. Unclear if app needs to be launched in between changing.

### Affected platforms

Windows

### Affected DevToys kind

DevToys (app with GUI)

### DevToys Version

Version 2.0-preview.1 | Windows 11 26100.712

### Relevant Assets/Logs

Exception from Visual Studio debugger:
`WebView2: Initialization failed due to a mismatch in DPI awareness. Please ensure that the DPI awareness of the host app matches the DPI awareness of current WebView2 processes using the same user data folder.`

**Comment by veler** [comment]
Hi,
Thanks for sharing. My machine is often in 125% too and I never encountered this issue unfortunately. 🤨
I would need more data. Perhaps you can share the logs by any chance? See https://devtoys.app/doc/articles/sysadmin/logs.html?tabs=windows

**Comment by BreeceW** [comment]
Sure, this is the log from when I first ran it. [log-20240203T122628.txt](https://github.com/user-attachments/files/15796392/log-20240203T122628.txt)
When I ran it in the debugger it did not get past `[1]	App is starting...` because it was stopping at the WebView2 initialization.
This is what the debugger output looked like. [native output.txt](https://github.com/user-attachments/files/15796412/native.output.txt)

I’m not certain it would fix the issue, but DevToys should declare per-monitor/per-monitor v2 scaling either way. Right now, it’s running in System scaling. It looks like WPF does not respect the declaration in the .csproj file and it has to be specified in the [SxS manifest](https://learn.microsoft.com/en-us/windows/win32/hidpi/setting-the-default-dpi-awareness-for-a-process#setting-default-awareness-with-the-application-manifest).
